specs/windows/windows_search.table

```diff
@@ -0,0 +1,13 @@
+table_name("windows_search")
+description("Run searches against the windows system index database.")
+schema([
+    Column("entity", TEXT, "index of entity"),
+    Column("attribute", TEXT, "attribute"),
+    Column("value", TEXT, "value"),
+    Column("query", TEXT, "windows search query", required=True),
+])
+implementation("system/windows/windows_search@genWindowsSearch")
+examples([
+    "select * from windows_search where query = 'select top 10 system.itempathdisplay from systemindex where scope = ''file:c:\'''",
```

Comment:
I've the feeling that this might overcomplicate the adoption of this table as users will have to know this additional SQL-like syntax to be able to use the table. This can be abstracted by using [ISearchQueryHelper](https://learn.microsoft.com/en-us/windows/win32/api/searchapi/nn-searchapi-isearchqueryhelper)

Users should be able to write queries just using [AQS](https://learn.microsoft.com/en-us/windows/win32/lwef/-search-2x-wds-aqsreference) keywords (which is a simplified way of performing advanced queries) and not have to include things such as the systemindex  

Another issue with this queries is that users end up owning the properties to display, which is bad IMO. Imagine an user forgetting to include  `system.itempathdisplay`, they will end up receiving information from items without context

Example queries that could be useful to support:

`select * from windows_search where query = "hello"`
this should return any files that matches `hello` keyword. User will receive the entire context (properties information) for the matched items

`select * from windows_search where query = "hello kind:rtf year:>2022"`
This AQS query sould return any files that matches `hello` keyword and were created from 2022 and on. User will receive the entire context (properties information) for the matched items

Reply:
now supporting AWS!